src/cli.rs

```diff
@@ -486,6 +486,25 @@ fn subcommand_scaffold() -> Command {
     ];
     vap_args.sort_by(|a, b| a.get_id().cmp(b.get_id()));
 
+    let mut admission_request_args = vec![
+        Arg::new("operation")
+            .long("operation")
+            .short('o')
+            .required(true)
+            .value_name("TYPE")
+            .value_parser(PossibleValuesParser::new(["CREATE"])) //TODO: add UPDATE and DELETE
+            .help("Kubewarden Custom Resource type"),
+        Arg::new("object")
+            .long("object")
+            .value_name("PATH")
+            .help("The file containing the new object being admitted"),
+        Arg::new("old-object")
```

Comment:
maybe worth commenting until we have UPDATE.

Reply:
I would like to do that with another PR, right after this one gets merged